JS/NodeFile.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express')
+const app = express();
+app.use(express.static('C:\\Users\\user\\WebstormProjects\\SaarZiv_OfekChat\\public'))
+
+var http = require('http').Server(app);
+var io = require('socket.io')(http);
+
+
+app.get('/', function (req, res) {
+	res.send('Hello World! !!!')
+});
+
+io.emit('chat message', { for: 'everyone' });
+
+var appUserId = 0;
+var userClass = function (){
+	this.userId = 0;
+	this.color = "";
+	this.fontWeight = "";
+	this.fontStyle = "";
+};
+
+io.on('connection', function(socket){
+	var userInstance = new userClass();
+	appUserId ++;
+	userInstance.userId = appUserId;
+
+	socket.on('disconnect', function(){
+		console.log('user disconnected');
+	});
+
+	socket.on('chat message',function (msg) {
+		var dividedColor = msg.split(" ");
```

Comment:
Think about the next stracture:

Instead of using regex for string, and instead of switch case:

actions = {
'\setColor':  function setColor(){},
'\setBold':  function setBold(){},
'\Etc..':  function doSomething(){},
}

and then:

actions[msg.split(" ")[0]]();